pkg/storaged/iscsi-panel.jsx

```diff
@@ -171,7 +167,7 @@
                                             username: true, // makes it red without text below
                                             password: _("Invalid username or password")
                                         };
-                                    return cockpit.reject(err);
+                                    return Promise.reject(err);
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18487-20230310-143650-f841503b-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)